utils/handle_status_command.go

```diff
@@ -16,8 +16,10 @@ type EthCallResponse struct {
 	Result  string `json:"result"`
 }
 
-func HandleStatusCmd(geth_url string) (string) {
+func HandleStatusCmd(geth_url string) (string, []string) {
```

Comment:
Add a comment explain what gets returned (at a glance I don't know what []string is)

Reply:
This should also return an error rather than just print it. Remember that the end user does not have access to logs